### PR TITLE
Logo

### PR DESCRIPTION
**Pull request by jonenst**
this fixes the gse startup.

I'd like to simplify the svg even more, will do it later. (Using ApplyTransform to remove the ugly transforms and get the same thing as the previous logo.)

### CODE REVIEW

File: gse-spi/src/test/java/com/powsybl/spi/PowsyblLogoTest.java [hunk 1]
```diff
@@ -0,0 +1,21 @@
...
+
+/**
+ * @author Jon Harper <jon.harper at rte-france.com>
+ */
+public class PowsyblLogoTest {
+
+    @Test
+    public void testPowsyblLogo() {
+        //The svg is read in the constructor
+        new PowsyblLogo();
```
Comment (mathbagu) [inline gse-spi/src/test/java/com/powsybl/spi/PowsyblLogoTest.java]: Does this code really test / assert something? Do you expect it doesn’t throw any exception?

Comment (jonenst) [reply gse-spi/src/test/java/com/powsybl/spi/PowsyblLogoTest.java]: The previous and new code throw ClassCastExceptions if the svg doesn't have a particular structure.

**Comment by jonenst** [comment]
@mathbagu I went for a simpler approach, I reverted and then modified the svg with a text editor. https://github.com/powsybl/powsybl-gse/pull/169/commits/fa7aaa9a2d6c4857a403a11a4540e613c35c2979 is a lot simpler now. I only changed the logo on the left and didn't touch the text. I didn't add IDs or anything. And tests check the usage of the logo for the startup of the gse now.

Do you want the code the code for the ids ? It's easy to add it on top of this.

**Comment by jonenst** [comment]
in my opinion you can merge this.
Also, are the CodeCoverage commits necessary or should I remove them ?

**Pull request merged**